wbobeirne/joule-extension
# added icon for active/inactive channel

Closes #116 

### Added Icons for active/inactive channels.

Checking channel.active with a ternary next to node alias. If true return green circle else return a red circle.

### Steps to Test

Can manually set to false to verify when the channel state goes inactive that red circle will be displayed.

![image](https://user-images.githubusercontent.com/13033037/50532549-7269d000-0ae0-11e9-8144-9829ad8faf4c.png)
